annual-reviews-author-date.csl

```diff
@@ -224,16 +260,43 @@
             <number variable="edition" form="ordinal"/>
             <label variable="edition" form="short"/>
           </group>
+          <text macro="status"/>
         </group>
       </else-if>
       <else-if type="article-journal" match="any">
         <text macro="container-title"/>
         <text macro="locator"/>
       </else-if>
+      <else-if type="paper-conference" match="any">
+        <choose>
+          <if variable="volume" match="any">
+            <text macro="container-title"/>
+            <text macro="locator"/>            
+          </if>
+          <else>
+            <text macro="container-title"/>
+            <text macro="locator"/>
+            <choose>
+              <if variable="publisher" match="any">
+                <group delimiter=". " prefix=". ">
+                  <text macro="publisher"/>
+                  <text macro="status"/>
```

Comment:
There is a conditional to test for 'volume' in both the locator macro and the container macro that controls the placement of status. The point is to accommodate the different delimiters needed for journals/journal-like proceedings and books/book-like proceedings.